addon/components/horizontal-bar-chart.js

```diff
@@ -40,6 +42,60 @@ export default ChartComponent.extend(FloatingTooltipMixin,
   // Layout
   // ----------------------------------------------------------------------------
 
+  /**
+  * Overrides values in addon/mixins/axis-titles.js
+  * @override
+  */
+  xAxisPositionX: Ember.computed('graphicWidth', 'labelWidthOffset', function(){
```

Comment:
See `Include the correct dependent keys for computed properties` https://addepar.atlassian.net/wiki/display/EN/Code+Review+of+Ember.js+projects